### ISSUE TITLE
Fix https://peric.github.io/GetCountries/ serving https

### ISSUE THREAD

**Issue by jsgoupil** [peric/GetCountries#33]
That site does not work :'(

```
Mixed Content: The page at 'https://peric.github.io/GetCountries/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://api.geonames.org/countryInfoJSON?username=dperic'. This request has been blocked; the content must be served over HTTPS.
```

**Comment by raramuridesign** [comment]
@jsgoupil I have added a new pull request, just waiting on @peric to merge this.

**Comment by peric** [comment]
@raramuridesign thanks! Closing this.

https://peric.github.io/GetCountries/ works again.
